src/i18n/de.json

```
@@ -88,6 +88,16 @@
     "step2": {
       "title": "Was willst du hinzufügen"
     },
+    "municipality_and_place": {
+      "title": "Waar vindt dit evenement of deze activiteit plaats?",
+      "municipality": {
+        "change": "Wijzigen"
+      },
+      "place": {
+        "choose_label": "Kies een locatie",
+        "placeholder": "Naam of adres"
+      }
+    },
```

```suggestion
    "municipality_and_place": {
      "title": "Waar vindt dit evenement of deze activiteit plaats?",
      "municipality": {
        "change": "Wijzigen"
      },
      "place": {
        "choose_label": "Kies een locatie",
        "placeholder": "Naam of adres"
      }
    },
```

We also need to translate these for `de` and `fr`